content/enterprise_influxdb/v1.9/administration/manage/users-and-permissions/authorization-api.md

```diff
@@ -551,6 +570,52 @@ curl --location-trusted --negotiate -u "admin:changeit" -s -v \
 <
 ```
 
+##### Add permissions to a role for all databases for a specific database
+
```

Comment:
So in text... it would read something like:
In this example, we want to grant "Read", "Write", and "DropData" to the user named "spectre" for the specific databases called "MyDatabase" within the cluster.

Reply:
also change the "Verify user permissions" example to match what you set the permissions to for the "phantom" user.